### PR TITLE
 Leaderboard setup project :- module 2 

### PR DESCRIPTION
**Pull request by yusufyusie**
### JavaScript project for the Leaderboard list app using webpack and ES6 features, notably modules
**In this milestone requirements**
**Project 1: basic page structure.**

- Created an npm project with webpack
- Wrote plain HTML markup with minimum styling
- Used ES6 modules, with import and export.
- Don't make API calls.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "hint": "^7.1.10",
+    "style-loader": "^3.3.3",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack-cli": "^5.1.4"
```
Comment (V-Blaze) [inline package.json:26]: - You have done great so far :+1: :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :+1: 
- [ ] Currently, you do not have the `webpack-dev-server` in your project, and its needed to open your project server in the browser, I'll suggest you add this to your devDepndencies by running this command  `npm install webpack-dev-server` :+1: :wave:

File: src/assets/style.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
+/* stylelint-disable */
```
Comment (V-Blaze) [inline src/assets/style.css]: - [ ] As required by Microverse, you are not meant to use any linter disable rule in your project. The linters are there to make sure your code follows the best practices, I'll suggest you remove this disable rule and fix the issue correctly. :+1: :+1:

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard App</title>
+    <!-- <link rel="stylesheet" type="text/css" href="./assets/style.css" /> -->
```
Comment (V-Blaze) [inline src/index.html]: - [ ] Following the best practices, you are not meant to leave unused code as a comment in your codebase. I'll suggest you remove this as it makes your codebase look a bit unprofessional. :+1: :small_airplane:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @yusufyusie  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Correct Git flow✔
-  Webpack setup correctly👋 
-  Readme looks professional✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @yusufyusie, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**